### PR TITLE
Enable utility feature

### PR DESCRIPTION
**Pull request by jumapaoA**
This is the initial implementation of the utility feature based on the requirements.

### CODE REVIEW

File: util/config.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+config1:abc
+
+config2:567
```
Comment (meme2k21) [inline util/config.txt:3]: this config should be 567
the old value was 123

File: util/feature.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+feature 1 implement
+
+feature 2 implement
```
Comment (meme2k21) [inline util/feature.txt:3]: let's add a bit more to the feature2 implementation. I think it should handle multiple weights

**Review by meme2k21 — CHANGES_REQUESTED** [review]

File: util/feature.txt (hunk 2 above)
Comment (jumapaoA) [reply util/feature.txt:3]: @meme2k21 see the latest commit for the implementation.

**Review by jumapaoA — COMMENTED** [review]
Modified code based on review

**Pull request merged**